### ISSUE TITLE
Issue with isolatedModules

### ISSUE THREAD

**Issue by juuso-vaadin** [vaadin-component-factory/keyboard-shortcut-manager#2]
Keyboard shortcut manager has issues when using inside a typescript project:

```
node_modules/@vaadin-component-factory/keyboard-shortcut-manager/src/KeyboardShortcutManager.ts:249:3 - error TS1205: Re-exporting a type when the '
--isolatedModules' flag is provided requires using 'export type'.
 
249 KeyboardShortcutManagerOptions
```

It still works when isolatedModules is set to false.

**Comment by niiyeboah** [comment]
Should be fixed with version [`23.3.0`](https://vaadin.com/directory/component/vcf-keyboard-shortcut-manager-flow/23.3.0) of KSM.

**Comment by niiyeboah** [comment]
Should be fixed with version [`23.3.0`](https://vaadin.com/directory/component/vcf-keyboard-shortcut-manager-flow/23.3.0) of KSM.
